docs/topics/proc_exposing-application-metrics-using-prometheus-with-vertx.adoc

```diff
@@ -105,54 +108,43 @@ import io.vertx.micrometer.backends.BackendRegistries;
 
 public class MyVertxApp extends AbstractVerticle {
 
+  public class CustomLauncher extends Launcher {
```

Comment:
This should be a separate class in its own file

Reply:
Just to confirm that I am understanding this correctly. I should have two separate code block examples. One for the custom Launcher class and one for the MyVertxApp class.

Reply:
Exactly